pkg/cloud/common/config.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	DefaultConfigKey = "cloud.conf"
+)
+
+func NoOpTransformer(source *corev1.ConfigMap, infra *configv1.Infrastructure) (*corev1.ConfigMap, error) {
+	// Keys might be different between openshift-config/cloud-config and openshift-config-managed/kube-cloud-config
+	// Always use "cloud.conf" which is default one across openshift
+	output := source.DeepCopy()
```

Comment:
What is the motivation to not modify the configmap in place? Are we expecting the configmap name/namespace or other to be changed, if so, why aren't we doing that here?

Reply:
I haven't changed anything functional here: this code was previously in `prepareSourceConfigMap` and I simply renamed the variable (`s/cloudConfCm/output/`). Happy to drop this if that's doable. However, note that we do change the name and namespace but that is done separately ([here](https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/fde5bd720458cf2593c75bad7dbc2c9b2d8de9ba/pkg/controllers/cloud_config_sync_controller.go#L177-L181))